packages/adapter-next/src/hooks/slice-create.ts

```diff
@@ -39,6 +39,36 @@ const createComponentFile = async ({
 		options,
 	});
 
+	const placeholder = `
+		Placeholder component for ${data.model.id} (variation: {slice.variation}) slices.
+		<br />
+		<strong>You can edit this slice directly in your code editor.</strong>
+		{/**
+		 * 💡 Use Prismic MCP with your code editor
+		 *
+		 * Get AI-powered help to build your slice components — based on your actual model.
+		 *
+		 * ▶️ Setup:
+		 * 1. Add a new MCP Server in your code editor:
+		 *
+		 * {
+		 *   "mcpServers": {
+		 *     "Prismic MCP": {
+		 *       "command": "npx",
+		 *       "args": ["-y", "@prismicio/mcp-server"]
+		 *     }
+		 *   }
+		 * }
+		 *
+		 * 2. Select Claude 3.7 Sonnet for better output (more for a recommendation)
```

Comment:
this line will get outdated soon and could be misleading as soon as there's something better out there

Reply:
Yes we will need to keep it up-to-date with our recommendation. But today we cannot let them use whatever, too big a difference.

For the sharing comment, the problem is that it's never fully the same format, how the comment is done, format etc. :/

I tried, and it was more complex than just updating each adapter in the file directly.